### DISCUSSION TITLE
tarteaucitronCustomText list all options

### DISCUSSION THREAD

**Discussion by Zulfurion** [AmauriC/tarteaucitron.js#1012] [General]
Hi,
is it possible to have a list for the "tarteaucitronCustomText" options please ?

**Comment by AmauriC** [comment]
Here: https://github.com/AmauriC/tarteaucitron.js/blob/master/lang/tarteaucitron.en.js ;)

**Reply by Zulfurion** [reply]
cool merci beaucoup

**Comment by Zulfurion** [comment]
ah et une dernière chose, il y a moyen de mettre un texte différent sur le bouton "Tout refuser" du popup et celui du panneau de gestion des cookies please ?

**Comment by AmauriC** [comment]
Via la traduction non mais par exemple en jQuery c'est possible:

```
jQuery(function() {
   jQuery('#tarteaucitron #tarteaucitronAllDenied').text('Tout refuser');
});
```
